packages/devextreme/js/ui/drop_down_button.js

```diff
@@ -384,12 +367,12 @@ const DropDownButton = Widget.inherit({
             selectedItemKeys: isDefined(selectedItemKey) && useSelectMode ? [selectedItemKey] : [],
             grouped: this.option('grouped'),
             groupTemplate: this.option('groupTemplate'),
-            keyExpr: this._getKey(),
+            keyExpr: this._dataController.key(),
             noDataText: this.option('noDataText'),
             displayExpr: this.option('displayExpr'),
             itemTemplate: this.option('itemTemplate'),
             items: this.option('items'),
-            dataSource: this._dataSource,
+            dataSource: this._dataController.getDataSource(),
```

Comment:
don't we wanna pass a dataController instead of ds? Or u gonna make it later?

Reply:
I expected to do this later.